ruby_03-professional_rails_applications/building-a-vps.markdown

```diff
@@ -308,7 +308,15 @@ them which application process to use when serving requests.
 ### Configuring NGINX
 
 First, we're going to need to know where to tell Passenger to look for an installed version of Ruby.
-Fortunately passenger includes a script for finding this information, which we can use like so:
+Fortunately passenger includes a script for finding this information. Prior to setting that up we'll need to know where our installation of Ruby is hiding:
+
+```sh
+which ruby
```

Comment:
we actually don't need this part, or the part above that says "Prior to setting that up we'll need to know where our installation of Ruby is hiding:"

The point is to use the passenger-config option instead of `which ruby` since we need to use the rvm wrapper instead of the ruby binary, so let's take out these lines